webxrlayers-1.bs

```diff
@@ -238,6 +238,20 @@ The <dfn attribute for="XRProjectionLayer">ignoreDepthValues</dfn> attribute, if
 is <code>false</code> it indicates that the content of the depth buffer attachment will be used by the 
 [=XR Compositor=] and is expected to be representative of the scene rendered into the layer.
 
+XRTextureType {#xrtexturetype}
+-------------
+The {{XRTextureType}} enum defines what type of texture is allocated.
+
+<pre class="idl">
+enum XRTextureType {
+  "texture",
+  "texture-array"
+};
+</pre>
```

Comment:
I have a very slight preference for re-using what WebGL already defined for us, but I can see that I'm in the minority reading through #39 and this _does_ make the validation simpler, so 🤷‍♀.